.github/CONTRIBUTING.md

```diff
@@ -1,64 +1,65 @@
-[issue]: https://github.com/PlaceholderAPI/PlaceholderAPI/issues/new?template=change_request_placeholderapi.md
-[Discord]: https://helpch.at/discord
-[homepage]: http://contributor-covenant.org
-[version]: http://contributor-covenant.org/version/1/4/
+[issue]: https://github.com/PlaceholderAPI/PlaceholderAPI/issues/new
+[discord]: https://helpch.at/discord
+[Code of Conduct]: https://github.com/PlaceholderAPI/PlaceholderAPI/blob/master/CODE_OF_CONDUCT.md
+[wiki]: https://github.com/PlaceholderAPI/PlaceholderAPI/blob/master/wiki
 
-# Contributing
+# Contributing Guidelines
+We welcome everyone to contribute towards the PlaceholderAPI Project, but doing so will require you to follow specific rules to keep a consistent and welcoming way of contributing.
 
-When contributing to this repository, please first discuss the change you wish to make via [issue] or through [Discord] with the owners of this repository before making a change. 
+If you have any questions about this contributing file, feel free to contact us on our [Discord Server][discord].
 
-Please note we have a code of conduct, please follow it in all your interactions with the project.
+## Issues
+Like any other Project can you encounter bugs or a feature is missing for you in the plugin.  
+For that can you open an [issue] to report a bug or suggest a new feature to be added.
```

Comment:
I don't see where this could be seen as a question... Like it tells you that you can open an issue for reporting bugs or suggesting features.
It's not like "Can you please open an issue to do this?" but rather "Please open an issue for this" in my eyes... But you know... Everyone reads different.